README.md

```diff
@@ -0,0 +1,5 @@
+1. sing up or sing in bitbucket
+2. generate ssh key (https://confluence.atlassian.com/bitbucket/set-up-an-ssh-key-728138079.html#SetupanSSHkey-ssh2SetupSSHonmacOS/Linux)
+3. sudo apt install git docker docker-compose curl pkg-config pandoc supervisor python3-pip virtualenv postgresql postgresql-server-dev-all qemu-user-static whois -y
```

Comment:
Use syntax highlight when insert code to readme, see https://guides.github.com/features/mastering-markdown/ for the reference